python/GafferSceneUITest/ParameterInspectorTest.py

```diff
@@ -878,6 +878,125 @@ def testShaderAndParameter( self ) :
 			edit = addBTweak
 		)
 
+	def testSingleConnection( self ) :
```

Comment:
These new tests seem like they wouldn't work without the next commit - perhaps they belong with it?